### ISSUE TITLE
[f. req] Option: Draw a Curved Line To Select

### ISSUE THREAD

**Issue by gitut2007** [cpriest/SnapLinksPlus#329]
This is  similar to https://github.com/cpriest/SnapLinksPlus/issues/326 . 
Can be very useful, eg in the imdb list of cast, to Copy, Open or Both [ https://github.com/cpriest/SnapLinksPlus/issues/328 ] 
 eg only the females. 
When the mouse line crosses the links, it turns green.
Both red and green traces are shown, until releasing the click, to make clear what was chosed an what not:
https://1fichier.com/?j6z4flx6js08nr7zslla
[idk why i can never drag/add a jpg here in Ghitub]

**Comment by cpriest** [comment]
Yeah, honestly I've run into this need before and I just make 5 quick little boxes to open the links I want, so much easier than trying to thread a needle between links.

**Comment by gitut2007** [comment]
Doing quick little boxes might work conveniently for opening, but for copying you would need [for making just 1 paste vs as many pastes as  little boxes] sort of an aditional 'multi-copy' app thas adds the successively copied, 
so 1 additional addon, + extra steps eg taking care  that the 'multi-copy' app  hadn't keep unwanted past pre-SnapLinks copies and that it  'flushes' the clipboard after SnapLinks operation. 
And if there are way more than5 links, drawing becomes even more and more convenient than various little boxes .

But i see that this is, or was thought mainly for opening, more for immediate browsing, copying  being an afterthought, at least so far.

**Comment by cpriest** [comment]
Yeah, I can see that would be pretty useless with trying to copy the URL of several links.  I'm not sure I'm up for solving this particular problem, but I'll consider it.

There have been requests in the past (and I know some other similar clones have it) to be able to "continue selection," whereby the first box stays shown and you can select another, or adjust the one (or two/three) that are already chosen, before executing the action.

Also something I've considered, but I don't have a lot of time any more to work on this project besides keeping it working as-is with evolving browser requirements.
